src/storage.rs

```diff
@@ -16,7 +16,16 @@ pub enum KeyPrefix {
     Code = 0x3,
     Storage = 0x4,
     RelayerEvmAddressMap = 0x5,
-    // EthConnector = 0x6,
+    EthConnector = 0x6,
+}
+
+/// Enum used to differentiate different storage keys used by eth-connector
+#[derive(Clone, Copy, BorshSerialize, BorshDeserialize)]
+pub enum EthConnectorStorageId {
+    Contract = 0x0,
+    FungibleToken = 0x1,
+    UsedEvent = 0x2,
+    StatisticsAuroraAccountsCounter = 0x3,
```

Comment:
It makes sense to start statistics Id from something like `0xF` or `0x10`. So if other fields related to the contract itself are added in the future, they will come in order and not going to be mixed with Ids for Statistics.

I'll create a PR for that.